### DISCUSSION TITLE
Welcome to sparql-12 Discussions!

### DISCUSSION THREAD

**Discussion by JervenBolleman** [w3c/sparql-dev#137] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TallTed** [comment]
*Another* introduction of myself?  I'll let the comment linked below, and my github profile, speak for me for now...

https://github.com/w3c/sparql-12/issues/136#issuecomment-772835694

**Reply by VladimirAlexiev** [reply]
> _Another_ introduction of myself? I'll let the comment linked below, and my github profile, speak for me for now...

I'm responding ot Ted here

**Reply by VladimirAlexiev** [reply]
@TallTed can you quote-reply to https://github.com/w3c/sparql-12/discussions/137?sort=new#discussioncomment-338643 ("continuing this thought") and let's see where it goes.
Hmmm, nope, this more specific link `https://github.com/w3c/sparql-12/discussions/137?sort=new#discussioncomment-338643` is rendered as a global link to #137 so you have to search by the text in quotes.

**Reply by TallTed** [reply]
> quote-reply to #137 ("continuing this thought")

One whole level of depth to the conversation... This is _not_ a very well implemented threaded discussion space!

**Comment by VladimirAlexiev** [comment]
But here's a new train of thought, so a new subthread

**Reply by VladimirAlexiev** [reply]
continuing this thought, here's a subcomment

**Reply by VladimirAlexiev** [reply]
And further sub-sub-comment. You go `... > Quote reply` then delete what you don't want to quote

**Comment by VladimirAlexiev** [comment]
I want a new OUTDENTED thread!!

**Comment by TallTed** [comment]
I am unimpressed.  To put it mildly.

_(ETA: This should have been a top-level reply, not a reply to a lower level of the discussion.)_

**Comment by gkellogg** [comment]
I think if GH had implemented something more like review comments on lines of a commit the "threaded" concept could have been more useful. The important thing is to keep things that are closely relevant together, and I really don't see how this accomplishes that.

**Comment by TallTed** [comment]
It's too bad there weren't decades-worth of other attempts at building threaded discussions for them to learn from before building their own wheel from a raw boulder...

Oh, wait.

**Comment by ericprud** [comment]
In case GH is still accepting input on this. Can someone propose a publicly-viewable link to a node model for them to follow, maybe a nice viewer for mailing list archives (like W3C mailing list archive interface, but reflective of the aesthetics of this millennium)?

**Reply by TallTed** [reply]
I care less about the aesthetics of this millennium (how much value do avatars alongside user handles really add? OK, take a *ahem* page from LiveJournal/DreamWidth code...), than I do about simple discussion tree/thread displays (again, see LiveJournal/DreamWidth).

Almost any reasonably recent NNTP reader would do better than this.  I seem to recall some attempts to reinvent NNTP with RSS feeds/readers, too.  But those are generally client-side readers, so I guess they're out of vogue again.

**Reply by ericprud** [reply]
Sure, I don't care about the aesthetics either, but feedback to github is going to go a lot further if they have a model they're motivated to copy.
